src/libs/ajax/WorkspaceDataService.test.ts

```diff
@@ -0,0 +1,134 @@
+import { asMockedFn } from '@terra-ui-packages/test-utils';
+import { Ajax } from 'src/libs/ajax';
+
+import { fetchWDS } from './ajax-common';
+
+type AjaxCommonExports = typeof import('src/libs/ajax/ajax-common');
+
+jest.mock('src/libs/ajax/ajax-common', (): AjaxCommonExports => {
+  return {
+    ...jest.requireActual('src/libs/ajax/ajax-common'),
```

Comment:
Nit: `jest.requireActual` takes a type parameter.

```suggestion
    ...jest.requireActual<AjaxCommonExports>('src/libs/ajax/ajax-common'),
```

Reply:
Fixed by a855291c176772dfb4dea24e4c8a82b0692f2042